src/lamplib/requirements.txt

```diff
@@ -12,4 +13,5 @@ shrub.py==1.1.0
 black==19.10b0
 setuptools==51.0.0
 pytest==6.0.2
+numpy==1.22.3  # again
```

Comment:
this can't be right?

Reply:
Another place where I lost my will to fight it. Ideally we'd just move the python to poetry.